doc/world_format.txt

```diff
@@ -365,7 +365,7 @@ if map format version <= 22:
     u16 content_size
     u8[content_size] content of metadata. Format depends on type_id, see below.
 if map format version >= 23:
-  u8 version (=1) -- Note the type is u8, while for map format version <= 22 it's u16
+  u8 version (=1 for map format version < 28, else =2) -- Note the type is u8, while for map format version <= 22 it's u16
```

Comment:
More readable suggestion: (line too long)
```
  u8 version -- Was `u16` for map format version <= 22
    -- Value `1` for map format version < 28
    -- Value `2` since map format version 28
```

Reply:
Fixed.